aws_change_observer/aws_change_observer_stack.py

```diff
@@ -31,6 +31,7 @@ def __init__(self, scope: Construct, construct_id: str, is_prod: bool = False, *
         DELETE_MARKER_REQUEST_LAMBDA_CODE_PATH = 'lambdas/delete_marker_request'
         UPDATE_MARKER_REQUEST_LAMBDA_CODE_PATH = 'lambdas/update_marker_request'
         OBSERVE_LAMBDA_CODE_PATH = 'lambdas/observe'
+        SNS_TOPIC_ARN = 'arn:aws:sns:us-east-1:123456789012:YourSNSTopic'  # Update this with your actual ARN
```

Comment:
@murauskas101 Since we're defining a stack and our architecture, no need for this. See here to create a topic.
https://docs.aws.amazon.com/cdk/api/v2/python/aws_cdk.aws_sns_subscriptions/README.html